src/org/loklak/server/Query.java

```diff
@@ -38,7 +38,7 @@
 
 public class Query {
     
-    private HttpServletRequest request;
+    public HttpServletRequest request;
```

Comment:
@Orbiter I did this because in https://github.com/loklak/loklak_server/pull/703/commits/e8917be11b71775fae7aa7ee27ac62a03bc91ef4 I needed to recover the baseurl in lines 80-82 so I did a query.request there. Is this ok?